opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -207,25 +207,8 @@ namespace Opm {
             }
         }
 
-        bool
-        isParallel() const
-        {
-    #if HAVE_MPI
-            if ( istlSolver().parallelInformation().type() !=
-                 typeid(ParallelISTLInformation) )
-            {
-                return false;
-            }
-            else
-            {
-                const auto& comm =boost::any_cast<const ParallelISTLInformation&>(istlSolver().parallelInformation()).communicator();
-                return  comm.size() > 1;
-            }
-    #else
-            return false;
-    #endif
-        }
-
+        bool isParallel() const
+        { return  grid_.comm().size() > 1; }
```

Comment:
then that's a dummy communicator, i.e. `Dune::CollectiveCommunication<Dune::No_Comm>`. this is also used for non-parallel grids.